lib/rtcpeerconnection/firefox.js

```diff
@@ -431,4 +366,17 @@ function overwriteWithInitiallyNegotiatedDtlsRole(description, dtlsRole) {
   return description;
 }
 
+/**
+ * Gets the active RTCRtpSenders of the RTCPeerConnection.
+ * @param peerConnection
+ * @returns {Map<Track.ID, RTCRtpSender>}
```

Comment:
It might be better to let the keys of this Map be MediaStreamTracks rather than MediaStreamTrack IDs, since the `addTrack` method works by checking if the MediaStreamTrack itself is already being sent, not just its ID; however, there's another complication: you can bypass the `addTrack` check if you use RTCRtpTransceiver or `replaceTrack` APIs. So I wonder: would it be better to have a `Map<MediaStreamTrack, Set<RTCRtpSender>>` here?

Reply:
@markandrus I think it would complicate matters if we also implemented for `replaceTrack`. And moreover we are not using it in our use case. If we simplified the multiple streams case based on the argument that we are not using it in our specific use case, I think we should be consistent here as well.

Reply:
OK, let's do that then, and perhaps include a note wherever we assume our WebRTC APIs are not being used in their full generality.